Dockerfile

```diff
@@ -1,38 +1,68 @@
-FROM ubuntu:16.04
+FROM ubuntu:latest
```

Comment:
Should this be explicitly `ubuntu:20.04`? Otherwise won't `latest` automatically become `22.04` at some point in the not too distant future?

Reply:
Yep! However, that would be fine since we are not reliant on the system version of python. It just means our build environment will be up-to-date and secure. 

It may mean a quick revisit to ensure that pyenv and any python dependencies don't break but we should be able to fix this quickly. If not, a quick change to fix the docker container back to ubuntu 20.04

Reply:
:+1: